mapie/regression/regression.py

```diff
@@ -513,12 +513,30 @@ def fit(
         MapieRegressor
             The model itself.
         """
-        fit_params = kwargs.pop('fit_params', {})
-        predict_params = kwargs.pop('predict_params', {})
-        if len(predict_params) > 0:
-            self._predict_params = True
-        else:
-            self._predict_params = False
+
+        # Initialize the estimator
```

Comment:
Remove comments from that section (if you think they are needed, it means that the functions name are not clear enough)